### PR TITLE
Add unit test for ImageInspect

### PR DESCRIPTION
**Pull request by vdemeester**
And… it's time for `ImageInspect` 🐻.

🐸

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: client/image_inspect_test.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+				return nil, err
+			}
+			return &http.Response{
+				StatusCode: http.StatusOK,
+				Body:       ioutil.NopCloser(bytes.NewReader(content)),
+			}, nil
+		}),
+	}
+
+	imageInspect, _, err := client.ImageInspectWithRaw("image_id", true)
```
Comment (duglin) [inline client/image_inspect_test.go]: Can we add a test for 'false' too?

Comment (vdemeester) [inline client/image_inspect_test.go]: Yes :stuck_out_tongue_closed_eyes:

Comment (calavera) [inline client/image_inspect_test.go]: don't promise if you cannot deliver @vdemeester :stuck_out_tongue:

Comment (vdemeester) [inline client/image_inspect_test.go]: Lol, will do, tomorrow :wink:

**Comment by calavera** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Pull request merged**